.github/workflows/nightlies.yml

```diff
@@ -0,0 +1,52 @@
+name: Upload nightly wheels to Anaconda Cloud
+
+on:
+  # Run daily at 1:23 UTC to upload nightly wheels to Anaconda Cloud
+  schedule:
+  - cron: '23 1 * * *'
+  # Run on demand with workflow dispatch
+  workflow_dispatch:
+
+jobs:
+  upload_nightly_wheels:
+    name: Upload nightly wheels to Anaconda Cloud
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.x'
+
+      # c.f. https://github.com/actions/download-artifact/issues/3#issuecomment-1017141067
+      - name: Download wheel artifacts from last build on 'main'
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          PROJECT_REPO=matplotlib/matplotlib
+          BRANCH=main
+          WORKFLOW_NAME="cibuildwheel.yml"
+          ARTIFACT_NAME=wheels
+
+          gh run --repo "${PROJECT_REPO}" list --branch "${BRANCH}" --workflow "${WORKFLOW_NAME}" --json status,databaseId > runs.json
```

Comment:
Is this guaranteed to a) be sorted the way you assume, and b) not include PRs that target the branch? It is not obvious from the `gh run list` docs.

Reply:
> Is this guaranteed to a) be sorted the way you assume

While I believe yes, in the RUN_ID line I explicitly do a sort in descending order of `databaseId` which is monotonically increasing in time so we don't have to hope.

> not include PRs that target the branch?

Hm. Good point! As I show in the "gh run example" in the PR body this can also pick up `pull_request` events. So I should rewrite this to explicitly filter on `event === "push"`.